### DISCUSSION TITLE
git@github.com: Permission denied (publickey).

### DISCUSSION THREAD

**Discussion by victor751** [community/community#50076] [Mobile] [answered]
### Select Topic Area

Question

### Body

How to fix this error when trying to add a repository to Github?

git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by maahaMaya** [comment, marked as answer]
reset your git access token 
[you can follow these LINK](https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)

**Reply by maahaMaya** [reply]
Can you please **mark as answered**

**Comment by Duongvankha** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> How to fix this error when trying to add a repository to Github?
> 
> git@github.com: Permission denied (publickey).
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.

Giggiu89u
